docs/send-data/opentelemetry-collector/data-source-and-configurations/additional-configurations-reference.md

```diff
@@ -0,0 +1,282 @@
+
+### Mapping OpenTelemetry concepts to Sumo Logic
+
+OpenTelemetry has a [rich data model](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto), which is internally constructed out of several layers. For all signals,
+these can be broken down into following:
+* **Resource**. Includes attributes describing the resource from which given set of data comes from. Should follow [resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions).
+* **Instrumentation Scope**. Additional information about the scope of data. For example, instrumentation library name.
+* **Record**. Refers to a specific entry of data, such as a Log, Span, or Metric. Each Record has its own set of attributes, which may include key/value pairs that are specific to the context of the Record. Some Record types may follow certain conventions for signal types, such as [trace](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions), [metrics](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions), or [logs](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/logs/semantic_conventions). Logs, in particular, can also include attributes in the body of the Record.
+
+As can be observed, while attributes can be present at both **Resource** and **Record level** currently, they are not created equal and should be interpreted separately. The Resource-Level attribute context is much broader, and they identify where data comes from, whereas Record-Level attributes concern just one record, often with many keys and values.
+
+At Sumo Logic, there is a concept of [Fields](https://help.sumologic.com/docs/manage/fields) for log data. Fields offer
+a powerful capability to associate indexable metadata with logs, though only limited number of them can be used at a given time. Also, they need to be defined first.
+
+Looking from the OpenTelemetry standpoint, [Fields](https://help.sumologic.com/docs/manage/fields) are a good match for the **Resource-level** attributes, while Log Record-level attributes are good fit for the [structured representation of the log via JSON](https://help.sumologic.com/docs/search/get-started-with-search/search-basics/view-search-results-json-logs), which
+is automatically supported by Sumo Logic Search.
+
+All **Resource-level** attributes are stored as fields. and any attributes that don't match a defined field will be skipped. You can check the list of ignored fields using the [dropped fields view](https://help.sumologic.com/docs/manage/fields/#view-dropped-fields). When a log contains attributes at the **Record-level** , they are stored as JSON, and if there is a body, it will be stored under the `log` key.
```

Comment:
```suggestion
All **Resource-level** attributes are stored as fields. and any attributes that don't match a defined field will be skipped. You can check the list of ignored fields using the [dropped fields view](https://help.sumologic.com/docs/manage/fields/#view-dropped-fields). When a log contains attributes at the **Record-level**, they are stored as JSON, and if there is a body, it will be stored under the `log` key.
```